### PR TITLE
More UI

### PR DESCRIPTION
**Pull request by NicolasWinsten**
* Monitor has ability to switch between screens
* Nematode display screen
* Nematode store screen
* Import/export nematode functionality
* Nematodes flash red when bitten

Temporary, use the arrow keys to switch screens for now

### CODE REVIEW

File: drawing.js [hunk 1]
```diff
@@ -204,34 +207,13 @@ class Canvas {
     //draw energy level bars above nematodes
     if(world.energyBarOn) {
       world.forEachNematode(n => this.DrawEnergyLevel(n));
-
     }
 
-    // draw the selected nematode's neural network and stats
     if (world.selectedNematode != null){
```
Comment (Brandon123b) [inline drawing.js:212]: Is there a reason you just removed all the comments without adding another comment? And why did you move some of it to a ticker?

File: nematode.js [hunk 2]
```diff
@@ -446,8 +482,9 @@ class Nematode {
         this.exists = false;
 
         // Clean un the neural network
-        this.nn.Destroy();
-        this.nn = null;
+        // The NN was not the culprit of the memory leak
```
Comment (Brandon123b) [inline nematode.js:482]: Even if the NN wasn't the culprit, Did you really need to remove this?

Comment (Brandon123b) [reply nematode.js:482]: I guess if it wasn't the issue, then it's fine without it. Just thought that it still wouldn't hurt to keep it.

File: NeuralNetwork/NeatNN.js [hunk 3]
```diff
@@ -17,6 +17,18 @@
 
 
 class NeatNN {
+    // TODO better way to map nodes to labels (easier UI and more generic)
```
Comment (Brandon123b) [inline NeuralNetwork/NeatNN.js]: This feels very messy and we should find a way to fix it.

File: nematode.js (hunk 2 above)
Comment (NicolasWinsten) [reply nematode.js:482]: The NN is required to exist in order to draw the NN display.  So a lot of refactor work would be needed to make the NN displays work with imported nematodes

File: drawing.js (hunk 1 above)
Comment (NicolasWinsten) [reply drawing.js:212]: The comments are removed because that's not what the code does anymore. It's just easier and cleaner if code is delegated to the right spots instead of lumped all together in this drawing.js loop

File: NeuralNetwork/NeatNN.js (hunk 3 above)
Comment (NicolasWinsten) [reply NeuralNetwork/NeatNN.js]: NeatNN should take input/output labels as arguments. It would've broke your training data

File: drawing.js (hunk 1 above)
Comment (Brandon123b) [reply drawing.js:212]: I just meant that there should be comments in the new code

File: NeuralNetwork/NeatNN.js (hunk 3 above)
Comment (Brandon123b) [reply NeuralNetwork/NeatNN.js]: How would it break the training data?

**Review by Brandon123b — APPROVED** [review]
